### DISCUSSION TITLE
Compatibility Issues with Crazyflie 2

### DISCUSSION THREAD

**Discussion by Dionysus7777777** [bitcraze/discussions#1485] [Q&A]
Is Crazyflie 2 also using the code from this library? If not, is there a library specifically provided for Crazyflie 2?

**Comment by knmcguire** [comment]
I've converted your issue to a discussion as that is the right place for these questions. This was an issue in the cfclient repository.

To answer your question @Dionysus7777777, the cfclient is indeed using code to control the crazyflie from a computer, but you can also use the crazyflie python library directly to control the crazyflie.

Please checkout the documentation: https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/
and the repository: https://github.com/bitcraze/crazyflie-lib-python
